test/wildcardEvents/addListener.js

```diff
@@ -333,6 +333,34 @@ module.exports = simpleEvents({
 
     test.expect(5);
     test.done();
-  }
+  },
+
+
+  '13. Use prepend on wildcards mode.': function (test) {
+
+    var emitter = new EventEmitter2({
+      wildcard: true
+    });
+
+    var type = ['some', 'listener', 'bar'];
+
+    const function1 = function () {
+      test.ok(true, 'The event was raised');
+    }
+
+    const function2 = function () {
+      test.ok(true, 'The event was raised');
+    }
+
+    emitter.on(type, function2);
+
+    emitter.prependListener(type, function1);
+
+    test.equal(emitter.listeners(type)[0], function1, 'Prepended listener is first');
+
+    test.expect(1);
+    test.done();
+
+  },
```

Comment:
The test is fine but uses the legacy nodeunit API adapter. It would be very helpful if you could refactor it a bit to use the assert lib as all our tests actually running with Mocha so there is no reason to keep writing legacy tests. Just import the `assert` module and write a plain Mocha test like [this one](https://github.com/EventEmitter2/EventEmitter2/blob/90a37aa060e37a24bdfa8e0cb543b7102782a26c/test/simple/symbol.js#L31-L43).

Reply:
Ok, i wrote prepend test in new file without changes in old test files. Look please, all is ok?